ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -23,6 +23,18 @@ import java.time.{Duration, Instant}
   "Errors raised by or forwarded by the Ledger API."
 )
 object LedgerApiErrors extends LedgerApiErrorGroup {
+  object UnsupportedFeature
+      extends ErrorCode(
+        id = "UNSUPPORTED_FEATURE",
```

Comment:
```suggestion
        id = "UNSUPPORTED_OPERATION",
```
align to internal name